lib/server.js

```diff
@@ -123,6 +142,16 @@ function create(log, error, config, routes, db) {
       }
     )
     server.auth.strategy(
+      // This strategy fetches the sessionToken with its
+      // verification state. It doesn't check that state.
+      'sessionTokenVerified',
```

Comment:
IIRC, the name incorporated by clients into their hawk credentials refers to the token type, not necessarily the specific auth strategy with which it's being used.  So a client making `"sessionToken"` credentials would work with any auth-strategy that uses sessionTokens, not just the auth-strategy named `"sessionToken"`.